content/concepts/consensus-network/amendments/amendments.md

```diff
@@ -41,8 +41,8 @@ Every 256th ledger is called a "flag" ledger. The process of approving an amendm
 
 The flag ledger itself has no special contents. However, during that time, the servers look at the votes of the validators they trust, and decide whether to insert an [`EnableAmendment` pseudo-transaction](enableamendment.html) into the following ledger. The flags of an EnableAmendment pseudo-transaction show what the server thinks happened:
 
-* The `tfGotMajority` flag means that support for the amendment has increased to at least 80% of trusted validators.
-* The `tfLostMajority` flag means that support for the amendment has decreased to less than 80% of trusted validators.
+* The `tfGotMajority` flag means that support for the amendment has increased to more than 80% of trusted validators.
+* The `tfLostMajority` flag means that support for the amendment has decreased to less than 80% (<=80%) of trusted validators.
```

Comment:
I think this wording is slightly easier to read:

```suggestion
* The `tfLostMajority` flag means that support for the amendment has decreased to 80% of trusted validators or less.
```

Reply:
Ye no problem with me, thanks